commands/help.rb

```diff
@@ -98,19 +98,25 @@ def self.help(command)
         If `-v` or `--verbose` is present, extra information will be displayed.
       EOT
     when 'search'
+      green = 'green'.lightgreen
+      red = 'red'.lightred
+      blue = 'blue'.lightblue
+      search_lib = 'crew search ^lib'.lightblue
+      search_audio = 'crew search audio'.lightblue
+      search_git = 'crew search -v git'.lightblue
       puts <<~EOT
         Look for package(s).
         Usage: crew search [-v|--verbose] [<pattern> ...]
         Both the name and description of packages will be searched.
-        If the package color is " + "green".lightgreen + ", it means the package is installed.
-        If the package color is " + "red".lightred + ", it means the architecture is not supported.
-        If the package color is " + "blue".lightblue + ", it means the architecture is supported but the package is not installed.
+        If the package color is #{green}, it means the package is installed.
```

Comment:
```
If the package color is #{'green'.lightgreen}, it means the package is installed.
```

Does this work?

Reply:
Yes but you were supposed to fix this days ago. :-P